### DISCUSSION TITLE
Audio live stream via BLE on ESP32

### DISCUSSION THREAD

**Discussion by redengineer14** [pschatzmann/arduino-audio-tools#1503] [Q&A] [answered]
Hi all. I'm looking for an ESP32 library that can support live streaming audio from the esp32 (collected via microphone) view BLE to be received by a computer to record the audio. 

Can someone please point me in the right direction to find what I'm looking for?

**Comment by pschatzmann** [comment, marked as answer]
I don't think BLE is part of the valid, supported options.

However, there are many other possibilities: just have a look at [these examples](https://github.com/pschatzmann/arduino-audio-tools/tree/main/examples/examples-communication).

Why BLE and what would be the operating system of your computer ?
